application/src/main/java/core/application/security/AuthenticatedUserService.java

```diff
@@ -0,0 +1,61 @@
+package core.application.security;
+
+import org.springframework.security.core.Authentication;
+import org.springframework.security.core.context.SecurityContextHolder;
+import org.springframework.stereotype.Service;
+
+import java.util.UUID;
+
+/**
+ * 인증된 사용자 정보를 제공하는 서비스 클래스
+ *
+ * 이 클래스는 현재 인증된 사용자의 ID, 이메일, 역할을 가져오는 메서드를 제공
+ * Spring Security의 SecurityContext에서 인증 정보를 읽어옴
+ */
+@Service
+public class AuthenticatedUserService {
+    /**
+     * 현재 인증된 사용자의 ID를 반환
+     *
+     * @return 인증된 사용자의 UUID, 인증되지 않은 경우 null 반환
+     */
+    public UUID getAuthenticatedUserId() {
+        Authentication authentication = SecurityContextHolder.getContext().getAuthentication();
+
+        if (authentication != null && authentication.getPrincipal() instanceof CustomUserDetails) {
+            CustomUserDetails customUserDetails = (CustomUserDetails) authentication.getPrincipal();
+            return customUserDetails.getUserId();  // UserId 반환
+        }
+        return null; // 인증되지 않은 사용자일 경우
+    }
+
+    /**
+     * 현재 인증된 사용자의 이메일을 반환
+     *
+     * @return 인증된 사용자의 이메일, 인증되지 않은 경우 null 반환
+     */
+    public String getAuthenticatedUserEmail() {
+        Authentication authentication = SecurityContextHolder.getContext().getAuthentication();
+
+        if (authentication != null && authentication.getPrincipal() instanceof CustomUserDetails) {
+            CustomUserDetails customUserDetails = (CustomUserDetails) authentication.getPrincipal();
+            return customUserDetails.getUserEmail();  // UserEmail 반환
+        }
+        return null; // 인증되지 않은 사용자일 경우
+    }
+
+    /**
+     * 현재 인증된 사용자의 역할을 반환
+     *
+     * @return 인증된 사용자의 역할 문자열, 인증되지 않은 경우 null 반환
+     */
+    public String getAuthenticatedRole() {
+        Authentication authentication = SecurityContextHolder.getContext().getAuthentication();
+
+        if (authentication != null && authentication.getPrincipal() instanceof CustomUserDetails) {
+            CustomUserDetails customUserDetails = (CustomUserDetails) authentication.getPrincipal();
+            return customUserDetails.getUserId().toString();  // UserRole 반환
```

Comment:
사용자의 역할을 반환한다 하셨는데, 

그럼 `getUserId` 가 아니라 `getUserRole` 를 호출해야 하지 않나요?

```suggestion
            return customUserDetails.getUserRole().toString();  // UserRole 반환
```

Reply:
준상님이 말씀해주신대로 잘못된 것이 맞습니다. 말씀해주셔서 감사합니다. 수정하도록 하겠습니다.

Reply:
https://github.com/prgrms-be-devcourse/NBE1_2_Team10/pull/64/commits/fd63c4ef904b3f0aaae64719b9924ca7a016b72e 커밋으로 수정됨

Reply:
c3a0f609bc29e9d5ff7e10eec2d9043249b0264a